app/channels/compatible_targets_channel.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class CompatibleTargetsChannel < ApplicationCable::Channel
+  def subscribed
+    stream_from current_user
```

Comment:
Don't you need to use `stream_for` here?

https://stackoverflow.com/questions/39002150/what-is-the-difference-between-stream-from-and-stream-for-in-actioncable

Reply:
I'll have a look.

Reply:
You're right https://guides.rubyonrails.org/action_cable_overview.html#streams